src/models/raster/ingestion/types.ts

```diff
@@ -96,3 +96,9 @@ export interface LayerData {
 export type NewRasterLayer = { metadata: NewRasterLayerMetadata } & LayerData;
 export type UpdateRasterLayer = { metadata: UpdateRasterLayerMetadata } & LayerData;
 export type IngestionUpdateJobParams = UpdateRasterLayer & { additionalParams: Record<string, unknown> };
+
+export interface FinalizeTaskParams {
```

Comment:
Please change the name of the interface to something like `IngestionNewFinalizeTaskParams` so it will indicate the job type because there are other 'finalize' tasks i.e export finalize.

Reply:
changed to  IngestionNewFinalizeTaskParams